lib/pixel.js

```diff
@@ -23,11 +23,12 @@ ColorString.colorValue = function colorValue(colors, g_table) {
   // correction table to correct the received value.
 
   // before sending, account for gamma correction.
-  colors[0] = g_table[colors[0]];
-  colors[1] = g_table[colors[1]];
-  colors[2] = g_table[colors[2]];
+  const leakProofColor = Object.assign({}, colors);
```

Comment:
So this all makes sense @hweeks - just a very small nit on the naming. I agree with the intent around scoping and stopping the leakage but wonder if we just call it `gammaAdjustedColours` or similar just to signal intent as to it's local purpose beyond the leakage issue.